ceph-releases/jewel/ubuntu/14.04/daemon/entrypoint.sh

```diff
@@ -376,7 +380,7 @@ EOF
     chmod +x /etc/service/${CLUSTER}-${OSD_ID}/run
   done
 
-exec /sbin/my_init
+exec /usr/bin/runsvdir -P /etc/service
```

Comment:
Hi Sebastian,

if I remeber correctly that is exactly the same path as in the my_init.
The problem is that if it gets called in my_init it was not working.
I have spend some hours on finding out why, but then I just called it
directly  because I don't see that anything of what my_init does besides
calling this is needed to start the osd processes.

2016-07-06 15:50 GMT+02:00 Leseb <notifications@github.com>:

> In ceph-releases/jewel/ubuntu/14.04/daemon/entrypoint.sh
> <https://github.com/ceph/ceph-docker/pull/318#discussion_r69733245>:
>
> > @@ -376,7 +380,7 @@ EOF
> >      chmod +x /etc/service/${CLUSTER}-${OSD_ID}/run
> >    done
> >
> > -exec /sbin/my_init
> > +exec /usr/bin/runsvdir -P /etc/service
>
> Why not changing the path inside the /sbin/my_init?
> I used to have that issue and at some point it was related to the path of
> the binaries.
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly, view it on GitHub
> <https://github.com/ceph/ceph-docker/pull/318/files/145c8417177f4380b344fa68a8b45c7b0a78dc04#r69733245>,
> or mute the thread
> <https://github.com/notifications/unsubscribe/ABrRNweNVNi4gH6S93wg4MsGLOXyFzAWks5qS7K1gaJpZM4I-c2B>
> .
>